apps/dashboard/app/globals.css

```
@@ -1,324 +1,124 @@
+.nav-list::before { content: "Workspace"; padding: 0 12px 8px; color: var(--muted-2); font-size: 10px; font-weight: 700; letter-spacing: .09em; text-transform: uppercase; }
+.nav-item { position: relative; width: 100%; min-height: 44px; display: flex; align-items: center; justify-content: space-between; padding: 0 12px 0 42px; border: 0; border-radius: 12px; color: #4e535b; background: transparent; cursor: pointer; text-align: left; transition: color .16s ease, background .16s ease, transform .16s ease; }
+.nav-item::before { position: absolute; left: 12px; width: 20px; height: 20px; display: grid; place-items: center; color: #747981; font-size: 14px; }
+.nav-item:nth-child(1)::before { content: "◫"; }.nav-item:nth-child(2)::before { content: "✓"; }.nav-item:nth-child(3)::before { content: "◇"; }.nav-item:nth-child(4)::before { content: "↗"; }.nav-item:nth-child(5)::before { content: "◉"; }.nav-item:nth-child(6)::before { content: "⌘"; }
+.nav-item:hover { color: var(--ink); background: var(--surface-2); transform: translateX(2px); }
+.nav-item.active { color: white; background: var(--base-blue); font-weight: 650; box-shadow: 0 8px 24px rgba(0,82,255,.18); }
+.nav-item.active::before { color: white; }
+.nav-item em { min-width: 26px; padding: 2px 6px; border-radius: 999px; color: var(--muted); background: #eceff2; font: 600 9px var(--font-geist-mono); font-style: normal; text-align: center; }
+.nav-item.active em { color: var(--base-blue); background: white; }
+.sidebar-foot { margin-top: auto; display: grid; gap: 13px; padding: 16px 16px 20px; border-top: 1px solid var(--line); }
+.preview-rail { width: fit-content; display: inline-flex; align-items: center; gap: 7px; padding: 6px 9px; border-radius: 999px; color: var(--amber); background: var(--amber-soft); font-size: 9px; font-weight: 750; letter-spacing: .07em; text-transform: uppercase; }
+.preview-rail i, .chain-dot, .healthy-dot, .halted-dot { width: 7px; height: 7px; display: inline-block; flex: 0 0 auto; border-radius: 50%; background: currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)